indexinglanguage/src/main/java/com/yahoo/vespa/indexinglanguage/expressions/GenerateExpression.java

```diff
@@ -82,7 +82,7 @@ public DataType setOutputType(DataType outputType, VerificationContext context)
             throw new VerificationException(this, "Generate expression requires either a string or array<string> output type, but got "
                     + outputType.getName());
         
-        super.setOutputType(null, outputType, null, context); // todo: Why not set actualOutput to outputType?
+        super.setOutputType(null, outputType, null, context); // TODO: Why not set actualOutput to outputType?
```

Comment:
It won't make any difference when required and actual is the same.
(The reason for separating them is just that the required type may be a supertype of the actually produced type.)